backend/src/main/java/de/christianporsch/backend/model/CardImage.java

```diff
@@ -0,0 +1,14 @@
+package de.christianporsch.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class CardImage {
+
+    private String normal;
```

Comment:
what means normal?

Reply:
normal is the size of the card. Its defined by scryfall, the website I have my bulkdata from.